marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataWithOptions.java

```diff
@@ -0,0 +1,89 @@
+package com.marklogic.hub.spark.sql.sources.v2;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.document.JSONDocumentManager;
+import com.marklogic.client.eval.EvalResultIterator;
+import com.marklogic.client.ext.util.DefaultDocumentPermissionsParser;
+import com.marklogic.client.ext.util.DocumentPermissionsParser;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.JacksonHandle;
+import org.apache.spark.sql.catalyst.InternalRow;
+import org.apache.spark.sql.sources.v2.writer.DataWriter;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class WriteDataWithOptions extends AbstractSparkConnectorTest {
```

Comment:
Put "Test" as a suffix on this class name, that's our convention for every test class.

Reply:
Renamed the class